tests/e2e/specs/shopper/mini-cart.test.js

```diff
@@ -554,7 +554,18 @@ describe( 'Shopper → Mini Cart', () => {
 
 			await expect( page ).toMatchElement( 'h1', { text: 'Checkout' } );
 
-			await expect( page ).toMatch( productTitle );
+			const orderSummaryToggle = await page.$(
+				'.wc-block-components-order-summary button[aria-expanded="false"]'
+			);
+
+			if ( orderSummaryToggle ) {
+				await orderSummaryToggle.click();
+			}
+
+			await expect( page ).toMatchElement(
+				'.wc-block-components-order-summary__content',
```

Comment:
Can these selectors be refactored somewhere else?

Reply:
@sunyatasattva Do you mean using another selector that indicates better that the current page is checkout?

Reply:
Ah, I think you mean 'scoping selectors', is that correct?

Reply:
Oh sorry, I meant to just put these selectors somewhere else like we do for the others. So no magic strings are within the code, and the code would look something like `await expect( page ).toMatchElement( selectors.orderSummary.content )`.